### PR TITLE
RR-269 - Wiremock stubs

### PR DESCRIPTION
**Pull request by mattwills-moj-digital**
Wiremock stubs to simulate Curious and CIAG induction service errors.

For example, this page is rendered when Curious is down:

image

Clearly these scenarios need looking at, which will require collaboration with Karen & Jo and separate stories to be created.

### CODE REVIEW

File: server/filters/formatAdditionalTrainingFilter.test.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ describe('formatLevelOfEducationFilter', () => {
       { source: 'MANUAL_HANDLING', expected: 'Manual handling' },
       { source: 'TRADE_COURSE', expected: 'Trade course' },
       { source: 'OTHER', expected: 'Other' },
-      { source: 'NONE', expected: 'None of these' },
+      { source: 'NONE', expected: 'None' },
```
Comment (mattwills-moj-digital) [inline server/filters/formatAdditionalTrainingFilter.test.ts:16]: (Unrelated change)

**Review by nathanrussell-moj-digital — APPROVED** [review]
A thing of actual beauty 👍
